pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.black]
-line-length = 140
+line-length = 100
```

Comment:
why shorten to 100? If we're going to shorten it why not follow PEP-8?

Reply:
I'm with you there. I'm pretty sure the black team did a huge amount of research on this, everything from accessibility to psychology to real-world usage stats, and an arbitrary value just invites more 🚲 'shedding when the whole promise of black is to liberate your team 🏇.